packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -149,16 +153,34 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
         resultHintString = FaceMatchStateStringMap[faceMatchState];
       }
 
+      // If the face is outside the oval set the aria-label to a string about centering face in oval
+      let a11yHintString = resultHintString;
+      if (faceMatchState === FaceMatchState.OFF_CENTER) {
+        a11yHintString = FaceMatchStateStringMap[faceMatchState];
+      }
+
+      // If the face match percentage reaches a 50% or 100% marks append it to the a11y label
+      if (faceMatchState === FaceMatchState.MATCHED) {
+        a11yHintString = `${a11yHintString}. 100% face fit.`;
+      } else if (faceMatchPercentage! > 50) {
+        a11yHintString = `${a11yHintString}. 50% face fit.`;
```

Comment:
Do these these need to be included in `displayText` for i18n?

Reply:
good call out, will add